src/Microsoft.Azure.WebJobs.Logging/Internal/LegacyHack.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace Microsoft.Azure.WebJobs.Logging
+{
+    // LogReader needs to handle older format.
+    // DO this in one central place so that we can remove this class when deprecate the old format. 
+    // One central place 
+    internal class LegacyHack
+    {
+        // The previous table name. 
+        public const string OldTableName = "AzureFunctionsLogTable";
+
+        //  null if not present
+        public static CloudTable GetLegacyTable(CloudTableClient client)
+        {
+            var table = client.GetTableReference(OldTableName);
+            if (!table.Exists())
+            {
+                return null;
+            }
+            return table;
+        }
+
+        // Does the suffix match the request to get hte legacy table? 
+        // Returns null if legacy table doesn't exist. 
+        public static CloudTable TryGetLegacy(CloudTableClient client, string suffix)
+        {
+            if (suffix == OldTableName)
+            {
+                return GetLegacyTable(client);
+            }
+            return null;
+        }
+
+        // -1 if not matched
+        public static long GetEpochFromTable(CloudTable table)
+        {
+            if (table.Name == OldTableName)
+            {
+                return 201609;
```

Comment:
September 2016 - suffix number for before version .7; used to keep things in order.